tasks/setup.js

```diff
@@ -0,0 +1,51 @@
+module.exports = function() {
+  // Global variables
+  var plugins = this.opts.plugins,
+      config  = this.opts.configs,
+      fs      = require('fs-extra'),
```

Comment:
Will do that tonight. Yeah it's quite a good library. They probably couldn't be bothered to do so. There are a lot of packages that provide the functionality fs-extra provides. Same for things like rimraf (which is actually included in fs-extra)